extensions/resteasy-reactive/quarkus-resteasy-reactive/common-runtime/src/main/java/io/quarkus/resteasy/reactive/server/common/Header.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.resteasy.reactive.server.common;
+
+public @interface Header {
```

Comment:
I think this and the other classes new classes should be in the same package as `@Cache`, `@RestForm` etc,